draft-ietf-tls-tls13.md

```diff
@@ -4557,7 +4612,14 @@ independent, so it is not feasible to compute one from another or
 the session secret from the exported value. Note: exporters can
 produce arbitrary-length values. If exporters are to be
 used as channel bindings, the exported value MUST be large
-enough to provide collision resistance.
+enough to provide collision resistance. The exporters provided
+TLS 1.3 are derived from the same handshake contexts as the
+early traffic keys and the application traffic keys respectively,
+and thus have similar security properties. Note that they do
+not include the client's certificate; future applications
+which wish to bind to the client's certificate may need
+to define a new exporter that includes the full handshake
+transcript.
```

Comment:
You are turning this into a wall of text.  There are several key pieces of information here: exporters produce unique secrets (1); exporters with different labels/contexts produce independent values (1); exporters can produce arbitrary output, but large values are needed to provide collision resistance which is important for channel bindings (3); the early exporter shares all the limitations of 0-RTT keys (4); exporters don't cover the client's certificate, only the client's fresh entropy (2).  I would arrange that as I have noted in parentheses.

Also, this section needs sub-sections.  It's a lot of information to wade through.